index.bs

```diff
@@ -94,13 +104,17 @@ ISSUE: Should this specification also define a structural brand stricter than 'm
 
 <h3 id="image-item">AV1 Image Item</h3>
 
-<p>When an item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>, it is called an <dfn>AV1 Image Item</dfn>, and SHALL obey the following constraints:
-<ul>
-<li>The item data SHALL be the content of a single valid [=AV1 Sync Sample=] as defined in [[!AV1-ISOBMFF]] with the additional constraints that there SHALL be only one [=Sequence Header OBU=] and it SHALL have its <code>[=still_picture=]</code> flag set to 1 and SHOULD have its <code>[=reduced_still_picture_header=]</code> flag set to 1.
-<li>The item SHALL be associated with an [=AV1 Item Configuration Property=].</li>
-</ul>
+When an image item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>, it is called an <dfn>AV1 Image Item</dfn>, and shall obey the following constraints:
+    
+  - The [=AV1 Image Item=] is a coded image item in which the coded image is a single valid [=AV1 Sample=] as defined in [[!AV1-ISOBMFF]].
+  - The [=AV1 Image Item=] shall be associated with an [=AV1 Item Configuration Property=].
 
-</p>
+The [=AV1 Sample=] that is the coded image of the [=AV1 Image Item=], is called <dfn>AV1 Image Item Data</dfn>. 
+The following constraints apply to [=AV1 Image Item Data=]:
+  - The [=AV1 Image Item Data=] shall be an AV1 sync sample, as defined in [[!AV1-ISOBMFF]].
```

Comment:
If you want to have a reference to [=AV1 Sample=], here it is fine. You could replace with:
"The [=AV1 Image Item Data=] shall be identical to the content of an [=AV1 Sample=] marked as 'sync', as defined in [[!AV1-ISOBMFF]].

Reply:
@cconcolato 
I changed the text as you requested, but this is what the AV1-ISOBMFF spec says:

> If an AV1 Sample is signaled as a sync sample (in the SyncSampleBox or by setting sample_is_non_sync_sample to 0), it SHALL be a Random Access Point as defined in [AV1].

Since in HEIEF we don't use the SyncSampleBox for image items, we don't "signal" samples as 'sync' or "mark" them as 'sync'. Maybe we should use the term "Random Access Point" instead?
E.g., "The [=AV1 Image Item Data=] shall be identical to the content of an [=AV1 Sample=] that is a Random Access Point, as defined in [[!AV1]]."

Anyway, I have changed the wording the way you requested, but we may want to revisit it later.